tests/plotting/test_plotting.py

```diff
@@ -49,10 +59,15 @@
 
 skip_not_vtk9 = pytest.mark.skipif(not VTK9, reason="Test requires >=VTK v9")
 
-# IMAGE warning/error thresholds (assumes using use_vtk)
+# Normal image warning/error thresholds (assumes using use_vtk)
 IMAGE_REGRESSION_ERROR = 500  # major differences
 IMAGE_REGRESSION_WARNING = 200  # minor differences
 
+# Image regression warning/error thresholds for releases after 9.0.1
+HIGH_VARIANCE_TESTS = ['test_pbr', 'test_set_viewup', 'test_add_title']
```

Comment:
We might consider defining this as a set, because we're only using it to check membership
```suggestion
HIGH_VARIANCE_TESTS = {'test_pbr', 'test_set_viewup', 'test_add_title'}
```

That being said, performance differences can be measured in nanoseconds so whether we make this change should be based purely on semantics:
```py
In [346]: HIGH_VARIANCE_TESTS = ['test_pbr', 'test_set_viewup', 'test_add_title']
     ...: %timeit 'test_set_viewup' in HIGH_VARIANCE_TESTS
     ...: %timeit 'test_potato' in HIGH_VARIANCE_TESTS
     ...: HIGH_VARIANCE_TESTS = set(HIGH_VARIANCE_TESTS)
     ...: %timeit 'test_set_viewup' in HIGH_VARIANCE_TESTS
     ...: %timeit 'test_potato' in HIGH_VARIANCE_TESTS
     ...: 
42 ns ± 0.76 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
68.8 ns ± 21.1 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
29.9 ns ± 0.526 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
31.7 ns ± 0.255 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

Reply:
Set makes more sense as there's no reason to have duplicate items.